app/models/user_model.py

```diff
@@ -0,0 +1,92 @@
+""" This is the Users' models file."""
+from ..utils.validation import Valid
+
+class Base:
```

Comment:
This name can be made Person. Base is not descriptive, if it wasn't for the comment I would not have known what It was about.